### DISCUSSION TITLE
Welcome to Docker for Node.js Discussions!

### DISCUSSION THREAD

**Discussion by BretFisher** [BretFisher/docker-mastery-for-nodejs#80] [Announcements]
## 👋 Welcome!
  This is a supplemental discussion area for my Udemy "Docker for Node.js Projects From a Docker Captain" course. Things to note:
  * Official Udemy course support (Q&A) is in the Udemy interface and checked most weekdays. It's still the best way to engage with the support team on course issues.
  * But, not a lot of *discussion* happens there. It's mostly just "ask the instructor and get an answer". It's not great for ideas, opinions, or long-form discussions outside of course lecture break/fix issues. Hopefully, this GitHub Discussions helps.
  * There's often great content in that Q&A that I'd like to make public, so I might recommend you move your Q&A from the course to this area, to illicit a wider community response.
  * If you prefer live chat, we're also active daily in the DevOps Discord (with channels for each course): https://devops.fan

By using these discussions, you agree with the following our [community code of conduct](https://github.com/BretFisher/udemy-docker-mastery/blob/main/slack-chat/coc.md). Organizers will enforce this code throughout all channels. We are expecting cooperation from all participants to ensure a safe environment for everybody.

TL;DR

* Be kind, patient, and excellent to each other

Enjoy! 🙌  🤜 🤛
